roles/local-images/tasks/deploy.yml

```diff
@@ -0,0 +1,55 @@
+# Always re-build on new namespace
+- name: Ensure namespace is absent before continuing...
+  k8s:
+    name: "{{ namespace }}"
+    api_version: v1
+    kind: Namespace
+    state: absent
+    wait: yes
+
+- name: Create namespace for local-images
+  k8s:
+    name: "{{ namespace }}"
+    api_version: v1
+    kind: Namespace
+    state: present
+
+- name: Build sample new-app using s2i
+  shell: "oc -n {{ namespace }} new-app {{ s2i_app_location }}"
```

Comment:
Just noticed, may you please change the `oc` to `{{ oc_binary | default('oc') }}`? We need it badly on CI side.

Reply:
I will address this one on a subsequent PR, I want to sync with mig-ci in how we address oc_binary location...